### DISCUSSION TITLE
Important packages, outside of Core

### DISCUSSION THREAD

**Discussion by Saeeed-B** [meteor/meteor#11964] [Compat Packages]
Greetings and devotion to serve all friends.

As a Meteor lover, I want this formwork to stay and excel.
But I think a key point can help Meteor survive and excel.

**Bring commonly used packages to the core :**
One of the biggest problems with node js frameworks is that they have a lot of external dependencies.
Especially when we have to meet our great needs with external packages.
External packages mean dependence on individuals,
The people who provide the packages may leave the package at one time.
On the other hand, in order to provide the basic needs, they must know the core of the formwork well. That sometimes is not the case.

**Let's save Meteor from this trap.**

Look realistic.
What is one of the keys to Flutter's success against  React Native?
Learning and getting started with  React Native is very simple and fast. But with further development, we realize that we need external packages for important tasks, and this is like entering a black hole.
**But Flutter delivered the important packages within its core, and it was bold.**

**Dependence within the kernel reassures the developer. And the developer is not worried about the future.**

what do you say ?

**Comment by StorytellerCZ** [comment]
There is an endless discussion going on which packages should and should not be in core. Meteor Community Packages helps to aleviate thsi problem a bit (and I'm planning improvements in this regard, stay tunned), but it comes to figuring out what the core Meteor should provide. With the accounts packages as an example this is quiet a bit net. The problem is though the bigger the core, the more resources it takes to develop and maintain.

**Reply by Saeeed-B** [reply]
> The problem is though the bigger the core, the more resources it takes to develop and maintain.

**Exactly I think Meteor is afraid of inserting packages into the `Core ` because it does not have the manpower to manage them.**

**Reply by Saeeed-B** [reply]
@StorytellerCZ  Rest assured, the more developers can trust Meteor, the more welcome it will be.
